mgorus.go

```diff
@@ -55,9 +55,9 @@ func NewHookerWithAuthDb(mgoUrl, authdb, db, collection, user, pass string) (*ho
 
 func (h *hooker) Fire(entry *logrus.Entry) error {
 	data := make(logrus.Fields)
-	data["Level"] = entry.Level
-	data["Time"] = entry.Time
-	data["Message"] = entry.Message
+	data["level"] = entry.Level.String()
```

Comment:
Why changed to lower case? This may be a break change.

Reply:
you're right, reverted to uppercase